drivers/scheduler/rke/rke.go

```diff
@@ -270,6 +270,53 @@ func (r *Rancher) SaveSchedulerLogsToFile(n node.Node, location string) error {
 	return err
 }
 
+// MoveNamespaceToNoProject moves namespace to no project
+func (r *Rancher) MoveNamespaceToNoProject(nsList []string) error {
```

Comment:
Can we call this function `RemoveNamespaceFromProject`?

Reply:
done